src/courses/delta/03.md

```diff
@@ -0,0 +1,87 @@
+---
+order: 03
+next: 04.md
+title: 3. Delta Process CLI Tools
+author: Daniel Medina, George Dias
+---
+
+## 3.1 CLI Tools
+
+The delta process consists of one or both of the following commands:
```

Comment:
I would probably introduce the SAF CLI first here. You could put its icon for visual interest and also link to how to install or use it so people know where to start to find the necessary commands. The idea is that someone could use this class as a How-to guide with no previous knowledge.